boards/pyboard/Makefile.include

```diff
@@ -9,5 +9,6 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
 include $(RIOTMAKE)/tools/serial.inc.mk
 
 # this board is flashed using DFU
-DFU_ARGS += --alt 0 -s 0x8000000
+DFU_USB_ID = 1d50:607f
+FFLAGS_OPTS = --reset
```

Comment:
why only keep `--reset` for pyboard?

Reply:
In master, this is the only board that uses it. I think @cladmi had concerns on not introducing behavior changes in other boards.

Reply:
Oh true this was the only board including  `dfu.inc.mk`